### PR TITLE
Using instanceof instead of in_array

### PR DESCRIPTION
**Pull request by arthurprogramming**
Changes proposed in this pull request:

Using instanceof in order to not have to specify every single exception, we can use an parent class like NoReportToNewrelicException and extends from it.

### CODE REVIEW

**Comment by Jalle19** [comment]
What exactly does this fix?

**Comment by arthurprogramming** [comment]
Using instanceof in order to not have to specify every single exception, we can use a parent class like NoReportToNewrelicException and extends from it.

File: src/NewRelicExceptionHandler.php [hunk 1]
```diff
@@ -35,36 +33,33 @@ public function __construct($ignoredExceptions = false)
...
     /**
      * @inheritdoc
      */
     public function report(Exception $e)
     {
-        if (!in_array(get_class($e), $this->ignoredExceptions)) {
-            $this->logException($e);
+        foreach ($this->ignoredExceptions as $ignored) {
+            if ($e instanceof $ignored) {
+                return false;
```
Comment (Jalle19) [inline src/NewRelicExceptionHandler.php]: Should this be return void instead? I don't remember the parent method signature...

Comment (arthurprogramming) [reply src/NewRelicExceptionHandler.php]: You're right, it's meant to return void, i've updated the PR.

**Comment by arthurprogramming** [comment]
Hey, man.

Anything else you need me to change?

**Comment by Jalle19** [comment]
Nope, thanks!

**Comment by Jalle19** [comment]
Would you mind making a pull request for master too?

**Comment by arthurprogramming** [comment]
> Would you mind making a pull request for master too?

Done. 
https://github.com/digiaonline/lumen-newrelic/pull/15
BR

**Pull request merged**